### PR TITLE
Log update vital feature remodified 

### PR DESCRIPTION
**Pull request by saloni0419**
## Proposed Changes
Updated the human-readable value for the pain scale.

- Fixes #7301 

Screenshot of the fix
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers @khavinshankar 

## Merge Checklist

- [x] removed the appearance of "normal" against 1-4 painscale

### CODE REVIEW

File: src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res [hunk 1]
```diff
@@ -55,13 +55,17 @@ let make = (
     }
   })
 
-  let getStatus = (min, minText, max, maxText, val) => {
-    switch (val >= min, val <= max) {
-    | (true, true) => ("Normal", "#059669")
-    | (true, false) => (maxText, "#DC2626")
-    | _ => (minText, "#DC2626")
-    }
+let getStatus = (min, minText, max, maxText, val) => {
```
Comment (khavinshankar) [inline src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res]:
```suggestion
let getStatus = (val) => {
```

File: src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res [hunk 2]
```diff
@@ -104,7 +108,7 @@ let make = (
                       | None => setState(prev => {...prev, scale: 0})
                       }
                     }}
-                    getLabel={getStatus(2.0, "Low", 4.0, "High")}
+                    getLabel={getStatus(0.0, "No pain", 5.0, "Unbearable")}
```
Comment (khavinshankar) [inline src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res]:
```suggestion
                    getLabel={getStatus}
```

**Review by khavinshankar — CHANGES_REQUESTED** [review]
Just a minor change requiried, removed unwanted params

**Comment by saloni0419** [comment]
@khavinshankar I have done the changes , please do tell if there is anything more to do

**Review by khavinshankar — APPROVED** [review]

**Review by rithviknishad — CHANGES_REQUESTED** [review]
1. Conflicting with ideas in https://github.com/coronasafe/care_fe/issues/7378 cc: @skks1212 
2. As per the issue, 0 and 5 only needs to have text. Rest all options does not need a text.
3. Issue not linked to the PR. (Ref: https://github.com/coronasafe/care_fe?tab=readme-ov-file#contributing-to-care)

**Comment by saloni0419** [comment]
@rithviknishad These changes were told to me by @khavinshankar . Should I change it or not. Please tell me
https://github.com/coronasafe/care_fe/pull/7301#discussion_r1513172595

**Comment by shivankacker** [comment]
@saloni0419 It appears new changes were requested in another issue, that were solved in a PR here https://github.com/coronasafe/care_fe/pull/7380, which conflicts with this PR.

Due to this, the current PR will need to be closed : (

Thank you for your contributions!

**Pull request closed without merging**